lib/heex_formatter/phases/tokenizer.ex

```diff
@@ -37,9 +37,10 @@ defmodule HeexFormatter.Phases.Tokenizer do
   end
 
   defp tokenize({:text, _line, _column, text}, acc) do
-    string = List.to_string(text)
-    {tokens, :text} = HTMLTokenizer.tokenize(string, "nofile", 0, [], [], :text)
-    {tokens |> Enum.reject(&line_break?/1) |> Enum.reverse(), acc}
+    string = IO.iodata_to_binary(text)
+
+    {tokens, _} = HTMLTokenizer.tokenize(string, "nofile", 0, [], [], :text)
+    {remove_line_breaks_on_tokens(tokens) |> Enum.reverse(), acc}
```

Comment:
```suggestion
    {tokens |> Enum.reject(&line_break?/1) |> Enum.reverse(), acc}
```

I'd suggest going back to the previous version since it is explicit enough on what is happening 🙌🏻

Reply:
done